pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/bestpractices/UnusedLocalVariableRule.java

```diff
@@ -41,12 +41,22 @@ public Object visit(ASTVariableDeclaration node, Object data) {
                 continue;
             }
 
-            if (usage.hasImageEqualTo(variableName)) {
+            if (equalsIgnoreCase(variableName, usage.getImage())) {
                 return data;
             }
         }
 
         addViolation(data, node, variableName);
         return data;
     }
+
+    private static boolean equalsIgnoreCase(String a, String b) {
```

Comment:
I think [`StringUtils.equalsIgnoreCase`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#equalsIgnoreCase-java.lang.CharSequence-java.lang.CharSequence-) is exactly this method

Reply:
Oh yes, I forgot, I can use StringUtils....